google-cloud-spanner-hibernate-dialect/src/main/java/com/google/cloud/spanner/hibernate/schema/RunBatchDdl.java

```diff
@@ -50,6 +61,10 @@ public boolean beforeTablesOnCreation() {
 
   @Override
   public String[] sqlDropStrings(Dialect dialect) {
-    return new String[] {"RUN BATCH"};
+    if (schemaAction == Action.UPDATE) {
+      return new String[]{};
```

Comment:
Why `UPDATE` doesn't need `RUN BATCH`?

Reply:
The AbstractSchemaMigrator triggers the [drop statement and then the create statement for each auxiliary object](https://github.com/hibernate/hibernate-orm/blob/master/hibernate-core/src/main/java/org/hibernate/tool/schema/internal/AbstractSchemaMigrator.java#L167), but this would cause us to produce the statements in this order:

```
RUN BATCH
START BATCH DDL
... all your ddl statements ...
RUN BATCH
```

So I make it so it does not produce a drop string so that it will not produce the extra RUN BATCH.

```
START BATCH DDL
... all your ddl statements ...
RUN BATCH
```